lms/djangoapps/verify_student/management/commands/send_verification_email_test.py

```diff
@@ -0,0 +1,78 @@
+"""
+Django admin command to send verification approved email to learners
+"""
+
+import datetime
+import logging
+import threading
+
+from crum import set_current_request
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.contrib.sites.models import Site
+from django.core.management.base import BaseCommand
+from django.test import RequestFactory
+
+from verify_student.emails import send_verification_approved_email, send_verification_confirmation_email
+from verify_student.toggles import use_new_templates_for_id_verification_emails
+
+logger = logging.getLogger(__name__)
+
+_thread_locals = threading.local()
+
+
+class Command(BaseCommand):
+    """
+    This command sends email to learner for which the Software Secure Photo Verification has approved
+
+    Example usage:
+        $ ./manage.py lms send_verification_email_test --username=staff --method=approved
+        OR
+        $ ./manage.py lms send_verification_email_test --username=staff --method=submitted
+    """
+    help = 'Send email to users for which Software Secure Photo Verification has expired'
+
+    def __init__(self, *args, **kwargs):
+        super(Command).__init__(*args, **kwargs)
+        self.request = RequestFactory().request()
+
+    def add_arguments(self, parser):
+        parser.add_argument('--username', help="The learner's email address or integer ID.")
+        parser.add_argument('--method', required=True, help="Choose email method. `approved` or `submitted`")
+
+    def init_request(self):
+        self.request.session = {}
+        self.request.site = Site.objects.get_current()
+        self.request.user = User.objects.get(username='staff')
```

Comment:
Why is this needed?

Reply:
This is required b/c internally when `is_flag_active` method is called it checks for the current request and if this isn't available then it would return `flag_undefined_default`
https://github.com/edx/edx-platform/blob/6ee2089077b76581e14f230f0c9224757dbdb652/openedx/core/djangoapps/waffle_utils/__init__.py#L281